app.js

```diff
@@ -442,6 +447,10 @@
 
       getEndDateQuery: function(queryType) {
         return new Date( this.$('.query.' + queryType + '.end-date').val().toLowerCase() );
+      }.bind(this),
+
+      stripHTMLTags: function(string) {
+        return string.replace(/<br>/, " ").replace(/<(?:.|\n)*?>/gm, '');
```

Comment:
Yeah, I thought it would make it more intuitive for searching things across lines.